web/templates/user_group_settings/cannot_deactivate_group_banner.hbs

```diff
@@ -1,3 +1,8 @@
-<p>
-    {{t "To deactivate this group, you must first remove all permissions assigned to it."}}
-</p>
+<div class="cannot-deactivate-group-banner main-view-banner error">
```

Comment:
This is not a main view banner but have still used these classes so that we can avoid duplicate CSS.